cocoa/test_model.py

```diff
@@ -0,0 +1,52 @@
+from unittest import TestCase
+
```

Comment:
@DinuBotan @danilashtefan please check this unit test and eventually if you think it's fine. After the merge you could use this pattern